feat(dailyReport): update daily report submission and disaster stats …

mehedi-hasan-2302 opened the pull request:
…aggregation

- Enhanced submitDailyReport in organizationService for detailed relief, rescue, and medical fields

- Updated getDisasterStats in coordinatorService and ReportRepository to include total relief items and medical aid

Warhammer4000 commented:
![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/pr-banner-default.svg)

### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/warhammer4000/pr-checks/14be9f2e-7798-4d47-9ae2-316847dd9990?refs=44136ca436399601d7928a06a20b9bb9f03cd2d4&source=prChecksComment)

mehedi-hasan-2302 committed:
feat(dailyReport): update daily report submission and disaster stats aggregation

- Enhanced submitDailyReport in organizationService for detailed relief, rescue, and medical fields

- Updated getDisasterStats in coordinatorService and ReportRepository to include total relief items and medical aid
